cmd/hubd/docker-compose-dev.yml

```diff
@@ -12,12 +12,12 @@ services:
       - HUB_ADDR_MONGO_URI=mongodb://mongo:27017
       - HUB_ADDR_IPFS_API=/dns4/ipfs/tcp/5001
       - HUB_ADDR_POWERGATE_API
-      - HUB_BUCKET_MAX_SIZE
-      - HUB_BUCKET_MAX_NUMBER_PER_THREAD
-      - HUB_BUCKET_TOTAL_MAX_SIZE
-      - HUB_THREAD_MAX_NUMBER_PER_OWNER
       - HUB_GATEWAY_SUBDOMAINS
       - HUB_EMAIL_SESSION_SECRET=hubsession
+      - HUB_BUCKETS_MAX_SIZE
```

Comment:
Note here @jsign: I changes these vars to be plural (buckets/threads vs bucket/thread) by changing the config key names to match the existing pattern (e.g., `addr.threads`). An annoying change :) but we haven't released the limits stuff yet so wanted to change before then.

Reply:
Np. Actually I was already using `latest` for the K8 deployment, so already used singular-names, but easy change.